src/indicator.js

```diff
@@ -89,6 +89,19 @@ const CPUFreqIndicator = new Lang.Class({
                 this._updateTurbo(true);
             }
         }
+
+        let result = GLib.spawn_command_line_sync(CPUFREQCTL + ' turbo get');
+        let returnCode = result[1];
+        this.isTurboBoostActive = returnCode;
+
+        result = GLib.spawn_command_line_sync(CPUFREQCTL + ' min get');
+        returnCode = result[1];
+        this.minVal = returnCode;
+
+        result = GLib.spawn_command_line_sync(CPUFREQCTL + ' max get');
+        returnCode = result[1];
+        this.maxVal = returnCode;
+
```

Comment:
I don't think this is neccessary. The values should be updated at this point anyway, by looking at the last settings file.

Reply:
I didn't see any logic around the last-settings file if installed globally (I am the Arch Linux maintainer for this package). I had played with a few different ideas but ultimately I didn't want to change any of the extension's behavior.